test/powershell/Modules/ThreadJob/ThreadJob.Tests.ps1

```diff
@@ -290,7 +304,7 @@ Describe 'Basic ThreadJob Tests' -Tags 'CI' {
 '@
 
         $result = & "$PSHOME/pwsh" -c $script
-        $result | Should -BeExactly "True"
+        $result | Should -BeTrue
```

Comment:
$result contains a string from an external application.
Please revert the change.